website/docs/r/org_oidc.html.markdown

```diff
@@ -0,0 +1,214 @@
+
+resource "vcd_org_oidc" "oidc" {
+  org_id                 = data.vcd_org.my_org.id
+  enabled                = true
+  prefer_id_token        = false
+  client_id              = "clientId"
+  client_secret          = "clientSecret"
+  max_clock_skew_seconds = 60
+  wellknown_endpoint     = "https://my-idp.company.com/oidc/.well-known/openid-configuration"
+
+  # Overrides:
+  access_token_endpoint = "https://my-other-idp.company.com/oidc/token"
+  userinfo_endpoint     = "https://my-other-idp.company.com/oidc/userinfo"
+}
+```
+
+Once the OIDC settings are created, if users want to restore an overridden value to the original one given by the
+well-known configuration endpoint, they must perform an update in code to set the previous value explicitly:
+
+```hcl
+data "vcd_org" "my_org" {
+  name = "my-org"
+}
+
+resource "vcd_org_oidc" "oidc" {
+  org_id                 = data.vcd_org.my_org.id
+  enabled                = true
+  prefer_id_token        = false
+  client_id              = "clientId"
+  client_secret          = "clientSecret"
+  max_clock_skew_seconds = 60
+  wellknown_endpoint     = "https://my-idp.company.com/oidc/.well-known/openid-configuration"
+
+  access_token_endpoint = "https://my-idp.company.com/oidc/token"          # Restores the previous value
+  userinfo_endpoint     = "https://my-other-idp.company.com/oidc/userinfo" # Still overridden
+}
+```
+
+In other words, removing the argument or setting it to `""` won't make the original value from the well-known configuration endpoint
+to be restored during updates.
+
+## Example Usage without Well-known Configuration Endpoint
+
+```hcl
+data "vcd_org" "my_org" {
+  name = "my-org"
+}
+
+resource "vcd_org_oidc" "oidc" {
+  org_id                      = data.vcd_org.my_org.id
+  enabled                     = true
+  prefer_id_token             = false
+  client_id                   = "clientId"
+  client_secret               = "clientSecret"
+  max_clock_skew_seconds      = 60
+  issuer_id                   = "https://my-idp.company.com/oidc"
+  user_authorization_endpoint = "https://my-idp.company.com/oidc/authorize"
+  access_token_endpoint       = "https://my-idp.company.com/oidc/token"
+  userinfo_endpoint           = "https://my-idp.company.com/oidc/userinfo"
+  scopes                      = ["openid", "profile", "email", "address", "phone", "offline_access"]
+  claims_mapping {
+    email      = "email"
+    subject    = "sub"
+    last_name  = "family_name"
+    first_name = "given_name"
+    full_name  = "name"
+  }
+  key {
+    id              = "rsa1"
+    algorithm       = "RSA"
+    certificate     = file("certificate.pem")
+    expiration_date = "2077-05-13"
+  }
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `org_id` - (Required) ID of the Organization that will have the OpenID Connect settings configured. There must be only one
+  resource `vcd_org_oidc` per `org_id`, as there is only one OpenID configuration per Organization
```

Comment:
Some bullet points use a dot at the end, others - don't. Let's make it consistent :)

Reply:
It may have slipped during re-check, but I couldn't find bullets with dot at the end (careful, as there are dots but mid-paragraph)

Reply:
True. GitHub review interface got me there :)